pygmt/tests/test_grdcontour.py

```diff
@@ -22,103 +21,88 @@ def fixture_grid():
     return load_earth_relief(registration="gridline")
 
 
-@check_figures_equal()
+@pytest.mark.mpl_image_compare
 def test_grdcontour(grid):
     """
     Plot a contour image using an xarray grid with fixed contour interval.
     """
-    fig_ref, fig_test = Figure(), Figure()
-    kwargs = dict(interval="1000", projection="W0/6i")
-    fig_ref.grdcontour("@earth_relief_01d_g", **kwargs)
-    fig_test.grdcontour(grid, **kwargs)
-    return fig_ref, fig_test
+    fig = Figure()
+    kwargs = dict(interval="1000", projection="W0/15c", frame=True)
+    fig.grdcontour(grid, **kwargs)
+    return fig
```

Comment:
No need to define the variable `kwargs`, because it's only used once.
```suggestion
    fig = Figure()
    fig.grdcontour(grid, interval="1000", projection="W0/15c", frame=True)
    return fig
```

Please also update other tests.